Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionLoggerManager.cs

```diff
@@ -322,21 +322,27 @@ public override void BusinessFlowStart(uint eventTime, BusinessFlow businessFlow
         public override void BusinessFlowEnd(uint eventTime, BusinessFlow businessFlow, bool offlineMode = false)
         {
             mContext.BusinessFlow = businessFlow;
-            Object BFR = mExecutionLogger.SetReportBusinessFlow(mContext, offlineMode, ExecutedFrom, this.Configuration.ExecutionLoggerConfigurationIsEnabled);
+            Object BFR = null;
 
-            if (this.Configuration.ExecutionLoggerConfigurationIsEnabled)
+            try
             {
-                if (this.ExecutedFrom == Amdocs.Ginger.Common.eExecutedFrom.Automation)
+                BFR = mExecutionLogger.SetReportBusinessFlow(mContext, offlineMode, ExecutedFrom, this.Configuration.ExecutionLoggerConfigurationIsEnabled);
```

Comment:
which peace of code was failing? mExecutionLogger.SetReportBusinessFlow? so BFR won't be Null again? let's talk on it on next meeting.